packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list.scss

```diff
@@ -0,0 +1,49 @@
+.fiat-transaction-list {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    margin: 0 auto;
+    width: 100vw;
+    max-width: 800px;
+
+    &__day {
+        display: flex;
+        flex-direction: column;
+
+        & > span {
```

Comment:
It's not a good idea to style the direct HTML tag. It's better to use classes.

Reply:
Done